deploy/pathplanner/settings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "robotWidth": 0.9,
```

Comment:
Change to accurate robot width and length (28 inches across, unless including bumpers, which would be 36.25in)